applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -94,7 +94,8 @@ void DistanceModificationProcess::CheckDefaultsAndProcessSettings(Parameters &rP
         "check_at_each_time_step"                : true,
         "avoid_almost_empty_elements"            : true,
         "deactivate_full_negative_elements"      : true,
-        "recover_original_distance_at_each_step" : false
+        "recover_original_distance_at_each_step" : false,
+        "variables_list"                  : ["PRESSURE","VELOCITY_X","VELOCITY_Y","VELOCITY_Z"]
```

Comment:
Is the name ok?

Reply:
I think it is not very descriptive. As an external user, I have no clue of why you need these variables in a process "for distance modification". However, I do not have a better suggestion. @rubenzorrilla any ideas?

Reply:
Some more suggestions:
"list_of_dofs_to_fix"
"list_of_variables_to_fix"
"variables_to_fix_list"
"dofs_to_fix_list"
"variables_to_fix"
"dofs_to_fix"

Reply:
Since this only applies for the `"deactivate_full_negative_elements": true` case, I'd rather call it `full_negative_elements_fixed_variables_list` or something similar that clearly states that these two arguments are tied.

Reply:
As a comment aside, I don't really like to specify each component one by one since all of them will be always fixed. However, no alternative comes to my mind (apart of creating a vector variables list, which I found it worse).

Reply:
I went for `full_negative_elements_fixed_variables_list` in the end.

Regarding your last comment, I didnt like it either so I added the possibility to define the whole vector, and then we check each component one by one.